system_tests/fast_confirm_test.go

```diff
@@ -40,9 +43,125 @@ import (
 	"github.com/offchainlabs/nitro/validator/valnode"
 )
 
+func TestFastConfirmationWithdrawal(t *testing.T) {
+	t.Parallel()
+	ctx, cancelCtx := context.WithCancel(context.Background())
+	defer cancelCtx()
+	builder, stakerA, cleanupBuilder, cleanupBackgroundTx := setupFastConfirmation(ctx, t)
+	defer cleanupBuilder()
+	defer cleanupBackgroundTx()
+
+	// Withdraw ETH from L2 to L1
+	arbSys, err := precompilesgen.NewArbSys(types.ArbSysAddress, builder.L2.Client)
+	Require(t, err)
+	authL2 := builder.L2Info.GetDefaultTransactOpts("User", ctx)
+	withdrawAmount := big.NewInt(1000)
+	authL2.Value = withdrawAmount
+	builder.L1Info.GenerateAccount("Receiver")
+	receiver := builder.L1Info.GetAddress("Receiver")
+	tx, err := arbSys.WithdrawEth(&authL2, receiver)
+	Require(t, err, "ArbSys failed")
+
+	receipt, err := builder.L2.EnsureTxSucceeded(tx)
+	Require(t, err)
+	if len(receipt.Logs) == 0 {
+		Fatal(t, "Tx didn't emit any logs")
+	}
+
+	// Wait for staker to confirm the withdrawal
+	time.Sleep(time.Second)
+	tx, err = stakerA.Act(ctx)
+	Require(t, err)
+	if tx != nil {
+		_, err = builder.L1.EnsureTxSucceeded(tx)
+		Require(t, err)
+	}
+
+	arbSysAbi, err := precompilesgen.ArbSysMetaData.GetAbi()
+	Require(t, err, "failed to get abi")
+	withdrawTopic := arbSysAbi.Events["L2ToL1Tx"].ID
+	authL1 := builder.L1Info.GetDefaultTransactOpts("User", ctx)
+	nodeInterface, err := node_interfacegen.NewNodeInterface(types.NodeInterfaceAddress, builder.L2.Client)
+	Require(t, err)
+	merkleState, err := arbSys.SendMerkleTreeState(&bind.CallOpts{})
+	Require(t, err, "could not get merkle root")
+	bridgeBinding, err := bridgegen.NewBridge(builder.L1Info.GetAddress("Bridge"), builder.L1.Client)
+	Require(t, err)
+	outboxAddress, err := bridgeBinding.AllowedOutboxList(&bind.CallOpts{}, big.NewInt(0))
+	Require(t, err)
+	outboxBinding, err := bridgegen.NewOutbox(outboxAddress, builder.L1.Client)
+	Require(t, err)
+	ouboxAbi, err := bridgegen.AbsOutboxMetaData.GetAbi()
+	Require(t, err, "failed to get abi")
+	outBoxTransactionExecutedTopic := ouboxAbi.Events["OutBoxTransactionExecuted"].ID
+	// Check logs for withdraw event
+	foundWithdraw := false
+	for _, log := range receipt.Logs {
+		if log.Topics[0] == withdrawTopic {
+			foundWithdraw = true
+			parsedLog, err := arbSys.ParseL2ToL1Tx(*log)
+			Require(t, err, "Failed to parse log")
+
+			// Check NodeInterface.sol produces equivalent proofs
+			outboxProof, err := nodeInterface.ConstructOutboxProof(
+				&bind.CallOpts{}, merkleState.Size.Uint64(), parsedLog.Position.Uint64(),
+			)
+			Require(t, err)
+			// Execute the transaction on L1
+			println("balance1: ", builder.L1.GetBalance(t, common.Address{}).Uint64())
```

Comment:
nitpick: how about using the log package from geth instead of println, so we have a more standardized output?